src/host_drivers.cmake

```diff
@@ -1,6 +1,6 @@
 # Building host device_control drivers here
 # I2C and SPI drivers are only built for PI
-if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL armv7l)
+if((${CMAKE_SYSTEM_PROCESSOR} STREQUAL "armv7l") OR (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64"))
```

Comment:
Previous test for Pi had (if UNIX and not APPLE) as well, consistency?

Reply:
I think actually we never hit this because we don't test on Mac. Should we guard against the possibility we might?